portfolio/src/main/webapp/index.html

```diff
@@ -6,7 +6,7 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body style="background-color:black;"></body>>
+        <body style="background-color:black;"></body>>
```

Comment:
Looks like you accidentally added some whitespace here, would you mind fixing it? It helps keep code easy to read if things stay aligned.